.github/workflows/maestro_test.yml

```
@@ -0,0 +1,166 @@
+
+      - name: Set gradle.properties
+        run: |
+          echo " " >> android/gradle.properties
+          echo "KEYSTORE=release.keystore" >> android/gradle.properties
+          echo "KEYSTORE_PASSWORD=${{ secrets.KEYSTORE_PASSWORD }}" >> android/gradle.properties
+          echo "KEY_ALIAS=${{ secrets.KEY_ALIAS }}" >> android/gradle.properties
+          echo "KEY_PASSWORD=${{ secrets.KEY_PASSWORD }}" >> android/gradle.properties
+
+      - name: Cat gradle.properties
+        run: cat android/gradle.properties
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)